### ISSUE TITLE
Missing env variables upon "Import Complete" trigger

### ISSUE THREAD

**Issue by wipika** [Sonarr/Sonarr#8165]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

According to the [Sonarr docs](https://wiki.servarr.com/sonarr/custom-scripts#on-importon-upgrade), the **sonarr_episodefile_sourcefolder** and **sonarr_episodefile_sourcepath** env vars should be exposed/passed to a custom script when the "On Import Complete" trigger is invoked.

However, these two env variables are absent, and only the following are provided by Sonarr with this trigger:

    sonarr_applicationurl
    sonarr_destinationfolder
    sonarr_destinationpath
    sonarr_download_client
    sonarr_download_client_type
    sonarr_download_id
    sonarr_episodefile_count
    sonarr_episodefile_episodeairdates
    sonarr_episodefile_episodeairdatesutc
    sonarr_episodefile_episodeids
    sonarr_episodefile_episodenumbers
    sonarr_episodefile_episodeoverviews
    sonarr_episodefile_episodetitles
    sonarr_episodefile_ids
    sonarr_episodefile_paths
    sonarr_episodefile_qualities
    sonarr_episodefile_qualityversions
    sonarr_episodefile_relativepaths
    sonarr_episodefile_releasegroups
    sonarr_episodefile_scenenames
    sonarr_episodefile_seasonnumber
    sonarr_instancename
    sonarr_release_group
    sonarr_release_indexer
    sonarr_release_quality
    sonarr_release_qualityversion
    sonarr_release_releasetype
    sonarr_release_size
    sonarr_release_title
    sonarr_series_genres
    sonarr_series_id
    sonarr_series_imdbid
    sonarr_series_originallanguage
    sonarr_series_path
    sonarr_series_tags
    sonarr_series_title
    sonarr_series_titleslug
    sonarr_series_tmdbid
    sonarr_series_tvdbid
    sonarr_series_tvmazeid
    sonarr_series_type
    sonarr_series_year
    sonarr_sourcefolder
    sonarr_sourcepath

### Expected Behavior

As per the documentation, **sonarr_episodefile_sourcefolder** and **sonarr_episodefile_sourcepath** env vars should be exposed

### Steps To Reproduce

1. Create a custom script as follows:

```
#!/usr/bin/env bash
set -euo pipefail

# Log helper – print to stderr so it shows up in Sonarr's logs
log() { printf '[envdump] %s\n' "$*" >&2; }

log "================ ENVIRONMENT VARIABLES ================"

# Print only variables that start with "sonarr_"
env | sort | grep -E '^sonarr_' >&2

log "================ END OF ENVIRONMENT ================"
```

2. set the script to be executable

3. Create a "Custom Script" connection under Settings > Connect, and set the trigger to "On Import Complete"
 
4. Initiate an episode download and inspect the system logs for the output.

5. The output should show the full list of environment variables passed from Sonarr to the script. The **sonarr_episodefile_sourcefolder** and **sonarr_episodefile_sourcepath** env vars are missing.

### Environment

```markdown
Version: 4.0.16.2944
Package Version: 4.0.16.2944-ls297 by linuxserver.io
.NET: Yes (6.0.13)
Docker Install: Yes
Using Reverse Proxy: No
Database: Sqlite 3.49.2
Database Migration: 217
```

### What branch are you running?

Main

### Trace Logs?

[sonarr.debug.txt](https://github.com/user-attachments/files/23422502/sonarr.debug.txt)
[sonarr.trace.txt](https://github.com/user-attachments/files/23422501/sonarr.trace.txt)
[sonarr.txt](https://github.com/user-attachments/files/23422503/sonarr.txt)

### Anything else?

<img width="714" height="859" alt="Image" src="https://github.com/user-attachments/assets/325eb21d-abb9-4b54-a974-504f3b31651c" />

**Comment by markus101** [comment]
As mentioned on reddit, these aren't missing, they're just not available as they target different things.

**Comment by wipika** [comment]
thank you @markus101 -- came back to close this. Indeed, switching the trigger to "On File Import" allows for these env vars to pass.
